spec.html

```diff
@@ -3720,6 +3731,8 @@
           </tbody>
         </table>
       </emu-table>
+      <!-- If ToObject is ever made hookable, we need to alert the editors of the HTML Standard as
+           that'll effect the security model of the Location exotic object. -->
```

Comment:
The problem is that the ECMAScript engine is updated by there is no notification going downstream. And then suddenly downstream has a security bug because they didn't pay attention to something they don't normally have to pay attention to.